libvast/src/logger.cpp

```diff
@@ -207,14 +213,41 @@ bool setup_spdlog(const vast::invocation& cmd_invocation,
   spdlog::init_thread_pool(queue_size, defaults::logger::logger_threads);
   std::vector<spdlog::sink_ptr> sinks;
   // Add console sink.
-  auto stderr_sink
-    = std::make_shared<spdlog::sinks::ansicolor_stderr_sink_mt>(log_color);
-  stderr_sink->set_level(vast_loglevel_to_spd(vast_console_verbosity));
+  std::string default_sink_type
+    = systemd::connected_to_journal() ? "journald" : "stderr";
+  auto sink_type
+    = caf::get_or(cfg_file, "vast.console-sink-type", default_sink_type);
+  auto console_sink = [&]() -> spdlog::sink_ptr {
+    if (sink_type == "stderr") {
+      auto stderr_sink
+        = std::make_shared<spdlog::sinks::ansicolor_stderr_sink_mt>(log_color);
+      return stderr_sink;
+    } else if (sink_type == "journald") {
+#ifndef VAST_ENABLE_JOURNALD_LOGGING
+      std::cerr << "Cannot use 'journald' sink, vast was built without systemd "
+                   "support\n";
```

Comment:
```suggestion
      std::cerr << "Cannot use 'journald' sink, VAST was built without systemd "
                   "support\n";
```

What's our policy regarding casing of error messages? I think we use lowercase in the logger, but what do we do on `cerr`?

Reply:
We also use lowercase to be consistent with all other uses of it.